release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -168,6 +168,11 @@ For more information, see xref:../installing/installing_aws/installing-restricte
 
 For more information, see the "Installation configuration parameters" section of the installation documentation for your platform.
 
+[id="ocp-4-11-heterogeneous-tech-preview"]
+==== {product-title} on Heterogeneous architectures (Technology Preview)
+{product-title} 4.11 introduces Heterogeneous architecture cluster support using Azure installer-provisioned infrastructures in Technology preview.
+This feature offers the ability to add  `arm64` worker nodes to an existing Azure installer-provisioned cluster with a heterogeneous installer binary as a day two operation. You can add `arm64` workers to your heterogeneous cluster by creating a custom Azure machine set that uses a manually generated `arm64` bootimage. Control planes on `arm64` architectures are not currently supported. For more information, see xref:../post_installation_configuration/deploy-heterogeneous-configuration.adoc[Configuring a heterogeneous cluster].
```

Comment:
maybe call out explicitly that the provisioned cluster must be x86 before adding the workers.